docs/source/guides/configuration.rst

```diff
@@ -14,6 +14,8 @@ at ``example/config.sample``
     "torii_port": 50051,
     "internal_port": 10001,
     "pg_opt": "host=localhost port=5432 user=postgres password=mysecretpassword dbname=iroha",
+    "working database": "iroha_working",
+    "maintenance database": "iroha",
```

Comment:
Despite you have introduced two more parameters with db names, there is still dbname=iroha in pg_opt parameter.

Unfortunately, it is not clear at all how should I manage these three parameters.